README.md

```diff
@@ -27,3 +28,19 @@ Include the Tide Site module in your composer.json file
 ```bash
 composer require dpc-sdp/tide_site
 ```
+
+# DRUSH
+The Drush command `tide-site-env-domain-update`, alias `tide-si-domup`, will
+update one or more taxonomy terms in the Sites vocabulary with new Domains.
+This command expects an environment variable `FE_DOMAINS` to exist. The var
+must be in this format:
+```
+FE_DOMAINS="4|develop.premier.vic.gov.au,174|temp.exmaple.com<br/>example.com,172|dddtemp.exmaple.com<br/>dsa.example.com"
```

Comment:
I know this has been hacked together so fit for purpose. It wouldn't have been much more effort though to use a proper yaml format... e.g. for readability and to avoid errors.
```
--- 
frontend_domains: 
  - 
    domains: 
      - develop.premier.vic.gov.au
      - temp.example.com
    id: 4
```

Reply:
Although it makes it hard to define it on the command line which is what you were aiming for I guess.